src/ios/SimpleCameraPreview.m

```diff
@@ -29,13 +29,17 @@ - (void) setOptions:(CDVInvokedUrlCommand*)command {
 }
 
 - (void) enable:(CDVInvokedUrlCommand*)command {
+    self.command = command;
     CDVPluginResult *pluginResult;
     if (self.sessionManager != nil) {
         pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Camera already started!"];
         [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
         return;
     }
 
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(sessionWasInterrupted:) name:AVCaptureSessionWasInterruptedNotification object:nil];
```

Comment:
https://stackoverflow.com/a/10272166. Tests this, we will be able to remove self.command